### ISSUE TITLE
Git: .husky/commit-msg: line 2: pnpm: command not found

### ISSUE THREAD

**Issue by unional** [microsoft/vscode#172883]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

1. setup husky and commit lint in a workspace and commit
2. It should work but does not.

It works find when using the integrated terminal.

```sh
2023-01-30 17:46:46.703 [info] > git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file - [29ms]
2023-01-30 17:46:46.703 [info] .husky/commit-msg: line 2: pnpm: command not found
```

It was working fine in the past (of course), but stopped working today.
I got this on private project. Will try to validate this in some open source projects.

**Comment by unional** [comment]
It's weird. It happens to one project and disabling all extension doesn't help.
Not just `pnpm` not found. `npm` also not found when I do `npm -v` in `.husky/commit-msg`

**Comment by unional** [comment]
Seems to be fixed after upgrading to 1.74.3
